testcases/main/POIFS/FileSystem/TestNPOIFSFileSystem.cs

```diff
@@ -110,10 +110,15 @@ protected static NPOIFSFileSystem WriteOutFileAndReadBack(NPOIFSFileSystem origi
         {
             FileInfo file = TempFile.CreateTempFile("TestPOIFS", ".ole2");
             using (FileStream fout = file.Open(FileMode.OpenOrCreate, FileAccess.ReadWrite))
+            try
             {
                 original.WriteFileSystem(fout);
-                original.Close();
             }
+            finally
+            {
+                fout.Close();
```

Comment:
Do we need to call `Close` explicitly here, when it was already managed by using statement?

Reply:
You are right. The code was just copied from POI, I will remove it.